src/ott/tools/plot.py

```diff
@@ -65,7 +65,9 @@ def __init__(
       cost_threshold: float = -1.0,  # should be negative for animations.
       scale: int = 200,
       show_lines: bool = True,
-      cmap: str = "cool"
+      cmap: str = "cool",
+      adjust_transparency: bool = False,
```

Comment:
I think this needs a better name, e.g., `scale_alpha_by_coupling`

Reply:
Thanks for the feedback, made the requested changes